src/Containers/Impl/Microsoft.R.Containers.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;net46</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;netstandard1.6;net46</TargetFrameworks>
```

Comment:
Why is this coreapp1.1;and standard1.6?

Reply:
15.5 wants references to match. I.e. something that is netcoreapp1.1 wants netcoreapp1.1 references or VS complains. Probably a bug, but annoying